server/server.go

```diff
@@ -59,6 +59,18 @@ const (
 	AuthenticationTLS
 )
 
+const (
+
+	//TLSVersion10 is a constant for tls version 1.0
+	TLSVersion10 = "1.0"
+	//TLSVersion11 is a constant for tls version 1.1
+	TLSVersion11 = "1.1"
+	//TLSVersion12 is a constant for tls version 1.2
+	TLSVersion12 = "1.2"
+	//TLSVersion13 is a constant for tls version 1.3
+	TLSVersion13 = "1.3"
```

Comment:
Do these need to be exported?
```suggestion
	tlsVersion10 = "1.0"
	tlsVersion11 = "1.1"
	tlsVersion12 = "1.2"
	tlsVersion13 = "1.3"
```

Reply:
@srenatus removed the constants altogether, instead I have added a map to map strings to tls Constants